### ISSUE TITLE
Ticket 2958 - Python console should have common readline shortcuts

### ISSUE THREAD

**Issue by luzpaz** [mumme74/FreeCAD#11]
Hey @mumme74 
Thanks for all your work on the debugger! I came across a ticket that possibly intersects with you work: https://freecadweb.org/tracker/view.php?id=2958 
What do you think? Is it something that you can address ?

**Comment by mumme74** [comment]
Yes that would probably be something I could do, I keep it in mind..

Sadly my time for development on FreeCAD is gonna have to be halted soon again for a couple of months.
Its going to be a very hectic autumn at work..
